deploy/azure/networking/vars.tf

```diff
@@ -226,13 +273,13 @@ variable "dns_zone_name" {
 ############################################
 
 variable "vmss_instances" {
-  default     = 1
+  default     = 2
   type        = number
   description = "Sets the default number of VM instances running in the VMSS."
 }
 
 variable "vmss_admin_username" {
-  default     = "adminuser"
+  default     = "rishi"
```

Comment:
will revert this back, but that would recreate the vmss so just finding the best time